pkg/controllers/job/job_controller_actions.go

```diff
@@ -507,19 +508,29 @@ func (cc *jobcontroller) createOrUpdatePodGroup(job *batch.Job) error {
 		return nil
 	}
 
-	if pg.Spec.MinMember != job.Spec.MinAvailable {
+	pgShouldUpdate := false
```

Comment:
Have you tested the effect of priorityClassName update on actions such as `preempt` ?

Reply:
I have tested allocate action， it‘s ok。 And all actions use priority(/priorityClassName) in similar ways, so theoretically there is no problem.